services/worker/src/worker/job_runners/split/opt_in_out_urls_scan_from_streaming.py

```diff
@@ -282,30 +296,24 @@ def compute_opt_in_out_urls_scan_response(
         )
     )
 
-    opt_in_urls = [
-        OptUrl(
-            url=urls[url_idx],
-            row_idx=url_idx // len(urls_columns),
-            column_name=urls_columns[url_idx % len(urls_columns)],
-        )
-        for url_idx in opt_in_urls_indices
-    ]
-    opt_out_urls = [
-        OptUrl(
-            url=urls[url_idx],
-            row_idx=url_idx // len(urls_columns),
-            column_name=urls_columns[url_idx % len(urls_columns)],
-        )
-        for url_idx in opt_out_urls_indices
-    ]
+    headers = ["url", "row_idx", "column_name", "opt_in", "opt_out"]
```

Comment:
`column_name` is clear, but we use the term `feature` in /row and /first-rows. Should we stick to `feature` / `feature_name`, or is it better to use `column` here?

Reply:
Also: is it OK to use `opt_in` and `opt_out` (which is what sampling.ai returns in its API (optIn/optOut)), or should we prefix with `is_` (or `opted_in`/`opted_out`) to hint that it's a boolean?

Reply:
I think we can leave as feature to keep consistency with firs_rows logic and also rename opt_x to is_opt_x (in/out)